sillsdev/web-languageforge
# upgrade LFNext to newly released sveltekit 1.0

## Description

This PR brings the LFNext app up to date with the recently released SvelteKit 1.0

### Type of Change

- dependency upgrade

## Checklist


## How to test

- Attempt to change password without providing anything and ensure error messaging looks good
- Attempt to change password while providing different passwords and ensure error messaging looks good
- View the project's dashboard and click the "show all" button at the bottom of the activity table

## qa.languageforge.org testing

Testers should add his/her findings to end of the PR in a comment and include screenshots, files, etc that are beneficial.
